README.md

```diff
@@ -1,290 +1,101 @@
-# Table of Contents
-* [What is cuTAGI ?](#What-is-cuTAGI)
-* [Python Installation](#pytagi-installation)
-* [C++/CUDA Installation](#cutagi-installation)
-* [Directory Structure](#directory-structure)
-* [License](#license)
-* [Related Papers](#related-papers)
-* [Citation](#citation)
-
-## What is cuTAGI ?
-cuTAGI is an open-source Bayesian neural networks library that is based on Tractable Approximate Gaussian Inference (TAGI) theory. cuTAGI includes several of the common neural network layer architectures such as full-connected, convolutional, and transpose convolutional layers, as well as skip connections, pooling and normalization layers. cuTAGI is capable of performing different tasks such as supervised learning, unsupervised learning, and reinforcement learning. The library includes some of the advanced features such as the capacity to propagate uncertainties from the input to the output layer using the the [full covariance mode for hidden layers](https://www.jmlr.org/papers/volume22/20-1009/20-1009.pdf), the capacity to estimate the [derivative](https://www.jmlr.org/papers/volume23/21-0758/21-0758.pdf) of a neural network, and the capacity to quantify heteroscedastic aleatory uncertainty. 
```

Comment:
"is based on Tractable" -> "is based on **the** Tractable"

Reply:
Done